client/src/components/projects/Project/Project.vue

```diff
@@ -124,6 +132,9 @@
       },
       technologies() :ITechnology[] {
         return this.$store.getters[PROJECT_TECHNOLOGIES_GROUPED];
+      },
+      isCompleted(): boolean {
+        return Boolean(this.project.enddate) && (new Date(this.project.enddate) <= new Date());
```

Comment:
cast to Boolean can be omitted if there's no TS error

Reply:
Oterwise error in component expected boolean got string if end date exists